src/Request.php

```diff
@@ -30,17 +30,23 @@ class Request
      */
     public function request($method, $url, $data = null)
     {
-        $url = Api::$baseUrl . $url;
+        $url = Api::getBaseUrl() . $url;
 
         if ($data === null)
             $data = array();
 
         $options = array(
```

Comment:
If we are doing a upgrade, let us drop support for EOL versions of PHP and start using square brackets?